Lars-02/Smoelenboek
# Test/searchbar

# Pull request for refactoring and testing searchbar.

## Description
Refactor and tests for searching on Roles (Functions), expertises, firstname and lastname.

## Testing
1. Open terminal
2. Run the command `php artisan test .\tests\Unit\SearchBarTest.php`. These should pass.
